sdk-installer/setup-git-sdk.bat

```diff
@@ -31,8 +31,8 @@
 
 @REM Avoid overlapping address ranges
 @IF 32 == @@BITNESS@@ @(
-	ECHO Auto-rebasing .dll files
-	CALL autorebase.bat
+    ECHO Auto-rebasing .dll files
+    CALL autorebase.bat
```

Comment:
Please drop this hunk, it is only changing white-space and that is orthogonal to the purpose of this PR.

Reply:
```suggestion
	CALL autorebase.bat
```